### ISSUE TITLE
LaTeX observation sink

### ISSUE THREAD

**Issue by dbenn** [AAVSO/VStar#296]
To generate table +/- plot.

**Comment by mpyat2** [comment]
So, it will be a 'publication-ready' plot? Exciting!

**Comment by dbenn** [comment]
Yes, exactly! I’ve been doing a lot of LaTeX recently, both leading up to a the 2.22.0 release and for study. Also, for example, pandas can generate LaTeX from data frames and I'm keen to do the same thing for VStar observations.

**Comment by dbenn** [comment]
Example LaTeX and PDF created by plugin:

- [jd.zip](https://github.com/AAVSO/VStar/files/10308218/jd.zip) (zipped LaTeX file)
- [VStar.pdf](https://github.com/AAVSO/VStar/files/10308206/VStar.pdf)

**Comment by dbenn** [comment]
Hi @mpyat2. See the example PDF (and LaTeX) generated from the first cut of the LaTeX observation sink.

There's still work to do but it's a start. Feedback welcome.

Merry Christmas mate. :)

**Comment by mpyat2** [comment]
Hi @dbenn , looks good so far!

**Comment by dbenn** [comment]
It may be worth considering generating observation files per series to simplify the LaTeX.

**Comment by dbenn** [comment]
Could also consider two plugins: one to generate a table and one to generate a plot. Instead I think that checkboxes in a single plug-in would be best.

**Comment by dbenn** [comment]
Also may need to allow generation of a table with ellipsis between head and tail rows, where head/tail row count is specifiable. Generating series files as mentioned above is also likely to be important.
